en/SetupView/Joystick.md

```diff
@@ -17,7 +17,7 @@ This is enabled by default for PX4 SITL builds (see the [Parameters](../SetupVie
 <!-- what is "Virtual RC by Joystick"? -->
 
 
-## Configuring the Joystick
+## Configuring the Joystick {#configure}
```

Comment:
double check the render here (nothing in github md, suppose gitbook takes care of it)

Reply:
I tested locally, it works.